src/containers/User/Collection/Gentks.tsx

```diff
@@ -19,57 +19,34 @@ import { InfiniteScrollTrigger } from "../../../components/Utils/InfiniteScrollT
 import { CardsContainer } from "../../../components/Card/CardsContainer"
 import { ObjktCard } from "../../../components/Card/ObjktCard"
 import { CardsLoading } from "../../../components/Card/CardsLoading"
-import { useCallback, useEffect, useMemo, useRef, useState } from "react"
+import {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react"
 import { useQuery } from "@apollo/client"
 import { Qu_userObjkts } from "../../../queries/user"
 import { CardSizeSelect } from "../../../components/Input/CardSizeSelect"
 import { GentksActions } from "./GentksActions"
 import { useQueryParamSort } from "hooks/useQueryParamSort"
+import { sortOptionsUserGentk } from "../../../utils/sort"
+import useFilters from "../../../hooks/useFilters"
+import { ListingFilters } from "../../../types/entities/Listing"
+import { getTagsFromFiltersObject } from "../../../utils/filters"
+import { DisplayToggle } from "./DisplayToggle"
+import { GentksList } from "./GentksList"
+import { UserContext } from "../../UserProvider"
 const ITEMS_PER_PAGE = 40
 
-const generalSortOptions: IOptions[] = [
-  {
-    label: "recently minted",
-    value: "createdAt-desc",
-  },
-  {
-    label: "oldest minted",
-    value: "createdAt-asc",
-  },
-  {
-    label: "recently bought",
-    value: "collectedAt-desc",
-  },
-  {
-    label: "rarity (rarest first)",
-    value: "rarity-asc",
-  },
-  {
-    label: "rarity (rarest last)",
-    value: "rarity-desc",
-  },
-]
-
-const searchSortOptions: IOptions[] = [
-  {
-    label: "search relevance",
-    value: "relevance-desc",
-  },
-  ...generalSortOptions,
-]
-
-function sortValueToSortVariable(val: string) {
-  if (val === "pertinence") return {}
-  const split = val.split("-")
-  return {
-    [split[0]]: split[1].toUpperCase(),
-  }
-}
-
 interface Props {
   user: User
 }
 export function UserCollectionGentks({ user }: Props) {
+  const { user: userLogged } = useContext(UserContext)
+  const [displayMode, setDisplayMode] = useState<"list" | "grid">("list")
```

Comment:
```suggestion
  const [displayMode, setDisplayMode] = useState<"list" | "grid">("grid")
```

i think we should set grid as default to avoid too much change in the UI